### PR TITLE
Added Follow unfollow

### PR DESCRIPTION
**Pull request by Rohanmond**
- Added follow unfollow Chips
- Changes in profile page

### CODE REVIEW

File: src/features/PostFeedPage/PostFeedPage.jsx [hunk 1]
```diff
@@ -71,7 +76,8 @@ export const PostFeedPage = () => {
     setPostInputForm({ ...postInputForm, pic: base64File });
   };
 
-  console.log(allPosts);
+  console.log(allUsers);
+  console.log(user);
```
Comment (znjs) [inline src/features/PostFeedPage/PostFeedPage.jsx:80]: can remove all console's

File: src/features/PostFeedPage/UserSlice.js [hunk 2]
```diff
@@ -1,3 +1,98 @@
...
+      thunkAPI.rejectWithValue(err.response.data);
+    }
+  }
+);
+
+export const followUser = createAsyncThunk(
+  'users/followUser',
+  async ({ followUserId, token, dispatch, userUpdateHandler }, thunkAPI) => {
+    try {
+      console.log(followUserId, token, dispatch, userUpdateHandler);
```
Comment (znjs) [inline src/features/PostFeedPage/UserSlice.js:30]: can remove the console

File: src/features/PostFeedPage/UserSlice.js [hunk 3]
```diff
@@ -1,3 +1,98 @@
...
+      state.isLoading = false;
+      ToastHandler(ToastType.Error, action.payload);
+    },
+
+    [followUser.pending]: (state) => {
+      state.isLoading = true;
+    },
+    [followUser.fulfilled]: (state, action) => {
+      state.isLoading = false;
+      console.log(action);
```
Comment (znjs) [inline src/features/PostFeedPage/UserSlice.js:75]: can remove the console

File: src/features/PostFeedPage/UserSlice.js [hunk 4]
```diff
@@ -1,3 +1,98 @@
...
+      state.isLoading = false;
+      ToastHandler(ToastType.Error, action.payload);
+    },
+
+    [unFollowUser.pending]: (state) => {
+      state.isLoading = true;
+    },
+    [unFollowUser.fulfilled]: (state, action) => {
+      state.isLoading = false;
+      console.log(action);
```
Comment (znjs) [inline src/features/PostFeedPage/UserSlice.js:88]: can remove the console

**Review by znjs — APPROVED** [review]
- Good PR
- Good naming conventions

**Review by romabulani — APPROVED** [review]
- Functionality is working as expected
- Good to merge
- Suggestions : Remove console.logs

**Review by ScoobyDubeyDo — APPROVED** [review]

**Pull request merged**